cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -100,10 +104,18 @@ func (cmd *command) run() error {
 
 	var err error
 	if cmd.K8s, err = kube.NewFromConfigWithTimeout("", cmd.KubeconfigPath, cmd.opts.Timeout); err != nil {
-		return errors.Wrap(err, "failed to initialize the Kubernetes client from given kubeconfig")
+		return fmt.Errorf("failed to initialize the Kubernetes client from given kubeconfig: %w", err)
+	}
+
+	if err := cmd.deploy(start); err != nil {
+		return err
+	}
+
+	if err := cmd.wizard(); err != nil {
+		return err
 	}
 
-	return cmd.deploy(start)
+	return cmd.wizard()
```

Comment:
Any specific reason why you call the wizard twice? Once should be enough.

I think you can remove the 
```
if err := cmd.wizard(); err != nil {
		return err
}
```

Reply:
This was a huge mistake! Thanks for catching it.